src/state/urls.js

```diff
@@ -23,3 +23,11 @@ export function getBigQueryURL(appName, appId, pingName, metricName) {
 export function getMetricSearchURL(app, search) {
   return `/apps/${app}?search=${search}`;
 }
+
+export function getDataCatalogMetricURL(appId, pingId, bigQueryColumn) {
+  const DATA_CATALOG_URL = "https://mozilla.acryl.io";
+  return `${DATA_CATALOG_URL}/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,moz-fx-data-shared-prod.${appId.replace(
+    /\./g,
+    "_"
+  )}.${pingId},PROD)/Lineage?column=${bigQueryColumn}`;
```

Comment:
It might be more useful to point to the `stable` or `live` version of this BigQuery table corresponding to the latest version since I believe that's what's mostly used in ETL downstream though I don't think that's consistent either. 

Also it looks like column lineage still doesn't support nested fields which these mostly are so the lineage view is empty when I click on it in the preview deploy. Maybe pointing to the top level column would be more useful? i.e. `metrics` instead of `metrics.x.y`

Reply:
> i.e. metrics instead of metrics.x.y

For clarification do you mean to link to the ping level instead?

Reply:
I mean the nested BQ columns but actually it might be fine. Here's an [example](https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1,PROD)/Lineage?column=%5Bversion%3D2.0%5D.%5Btype%3Dstruct%5D.%5Btype%3Dstruct%5D.metrics.%5Btype%3Dstruct%5D.uuid.%5Btype%3Dstring%5D.legacy_telemetry_client_id&filter_degree___false___EQUAL___0=1&page=1&unionType=0) with downstream column lineage that's okay. The first paragraph might still be relevant though (looking at `_stable`).

Reply:
Thanks for the clarification. 

> I mean the nested BQ columns but actually it might be fine. Here's an example with downstream column lineage that's okay.

Yea I saw some that worked and some that didn't show anything. Will leave it as is for now.

> The first paragraph might still be relevant though (looking at _stable).

I added:
- `_stable` and `_v1` with the assumption that's the latest version for most tables. Would this work?

Reply:
> `_stable` and `_v1` with the assumption that's the latest version for most tables. Would this work?
LGTM 👍 would be nice if we could use some metadata to determine the version, not sure how much work that is  here though